OpenEdXMobile/src/main/java/org/edx/mobile/model/course/CourseComponent.java

```diff
@@ -613,6 +613,21 @@ public CourseComponent getFirstIncompleteComponent() {
         }
     }
 
+    @Nullable
```

Comment:
Can we update the existing available method as follows:
```
    @Nullable
    public CourseComponent getFirstIncompleteComponent(boolean isVideoMode) {
        List<CourseComponent> components = isVideoMode ? getVideos(false) : children;
        for (CourseComponent block : components) {
            if (!block.isCompleted()) {
                return block;
            }
        }
        if (components.size() > 0) {
            return components.get(components.size() - 1);
        } else {
            return null;
        }
    }

Reply:
we can do so but increase the change set and in other places, we don't need video components.

Reply:
the change will affect only a single file.